inference-engine/thirdparty/clDNN/src/network.cpp

```diff
@@ -208,6 +209,28 @@ static void log_memory_to_file(memory::ptr mem, stream& stream, std::string laye
     (void)layerName;
 }
 #endif
+
+static bool is_dumped_layer(std::vector<std::string>& dump_layers, const primitive_id id) {
+    if (dump_layers.empty()) return true;
+    auto iter = std::find_if(dump_layers.begin(), dump_layers.end(), [&](const std::string& dl){
+        return (id.find(dl) != std::string::npos);
+    });
+    return (iter != dump_layers.end());
+}
+
+static std::vector<std::string> get_dump_layers(const std::string& str_dump_layers) {
+    std::vector<std::string> dump_layers;
+    std::stringstream ss(str_dump_layers);
+    std::string layer;
+    while (ss >> layer) {
+        dump_layers.push_back(layer);
+    }
+    return dump_layers;
+}
+
+static size_t iter_idx = 0;
+static std::vector<std::string> dump_layers;
```

Comment:
can we move this to debug_configuration so that we could get rid of redundant static global?

Reply:
Okay I see I will remove them